src/krati_Shukla/Number.java

```diff
@@ -0,0 +1,48 @@
+package krati_Shukla;
+
+public class Number {
+	
+	void displayEven(int no) {
+		System.out.println("Even numbers are : ");
+		for (int i=no; i<=15; i=i+2) {
+			System.out.println(i);
+		}
+	}
+		
+	void displayDiv1(int no) {
+		System.out.println("Numbers divisble by 5 are :");
+		for (int i = no; no <= 30; no = no+1) {
```

Comment:
You should always start with 1 to no. or can be 2 parameters - startNum and endNum
void displayDivNumber(int startNum, int endNum){
if(int num = startNum; num<=endNum; num++)
}